Sources/PrimerSDK/Classes/Core/Primer/Primer.swift

```diff
@@ -175,6 +176,76 @@ public class Primer {
             }
         }
     }
+    
+    public func showUniversalCheckout(on viewController: UIViewController, clientToken: String? = nil) {
+        Primer.shared.flow = .default
+        presentingViewController = viewController
+        
+        DispatchQueue.main.async { [weak self] in
+            guard let self = self else { return }
+            self.root = RootViewController()
+            guard let root = self.root else { return }
+            let router: RouterDelegate = DependencyContainer.resolve()
+            router.setRoot(root)
+            viewController.present(root, animated: true)
+        }
+    }
+    
+    public func showVaultManager(on viewController: UIViewController, clientToken: String? = nil) {
+        Primer.shared.flow = .defaultWithVault
+        presentingViewController = viewController
+        
+        DispatchQueue.main.async { [weak self] in
+            guard let self = self else { return }
+            self.root = RootViewController()
+            guard let root = self.root else { return }
+            let router: RouterDelegate = DependencyContainer.resolve()
+            router.setRoot(root)
+            viewController.present(root, animated: true)
+        }
+    }
+    
+    public func showPaymentMethod(_ paymentMethod: ConfigPaymentMethodType, on viewController: UIViewController, with clientToken: String? = nil) {
```

Comment:
[Feedback]
We need a flag for vault vs. non vault :)
"showPaymentMethod" can be for vaulting or for checkout 👍

Reply:
`intent` (checkout or vault) has been added